src/test/resources/fixtures/createLegacyBeaconRequest.json

```
@@ -0,0 +1,150 @@
+{
+  "data": {
+    "type": "legacyBeacon",
+    "attributes": {
+      "beacon": {
+        "pkBeaconId": 6062,
+        "statusCode": "ACTIVE",
+        "isWithdrawn": "N",
+        "isPending": "N",
+        "departRefId": "1187/02",
+        "hexId": "9D0E1D1B8C00001",
+        "manufacturer": "ACR",
+        "beaconType": "EPIRB",
+        "model": "CSTA 107, RLB 32 Cat 2 (Serialised non-TAC)",
+        "protocol": "None",
+        "serialNumber": 1763,
+        "cospasSarsatNumber": 476899,
+        "manufacturerSerialNumber": "None",
+        "coding": "None",
+        "firstRegistrationDate": "2002-09-06T00:00:00",
+        "lastServiceDate": "None",
+        "batteryExpiryDate": "None",
+        "withdrawnReason": "None",
+        "isArchived": "N",
+        "createUserId": 2889,
+        "createDate": "2015-03-24T00:00:00",
+        "updateUserId": 2889,
+        "updateDate": "2004-10-13T00:00:00",
+        "versioning": 0,
+        "note": "CHANGE OF ADDRESS/EMERGENCY DETAILS ADDED. 13/10/04\\r\\nKAYCEE"
+      },
+      "uses": [
+        {
+          "pkBeaconUsesId": 6057,
+          "fkBeaconId": 6062,
+          "vesselName": "KAYCEE",
+          "homePort": "None",
+          "maxPersons": 10,
+          "officialNumber": "None",
+          "rssSsrNumber": "None",
+          "callSign": "VQAS7",
+          "imoNumber": "None",
+          "mmsiNumber": 235007399,
+          "fishingVesselPln": "None",
+          "hullIdNumber": "None",
+          "cg66RefNumber": "None",
+          "aodSerialNumber": "None",
+          "principalAirport": "None",
+          "bit24AddressHex": "None",
+          "aircraftRegistrationMark": "None",
+          "areaOfUse": "None",
+          "tripInfo": "None",
+          "rigName": "None",
+          "beaconPosition": "None",
+          "position": "None",
+          "localManagementId": "None",
+          "beaconNsn": "None",
+          "beaconPartNumber": "None",
+          "notes": "None",
+          "pennantNumber": "None",
+          "aircraftDescription": "None",
+          "survivalCraftType": "None",
+          "communications": "VHF/DSC",
+          "isMain": "Y",
+          "createUserId": 2889,
+          "createDt": "2015-03-24T11:35:11",
+          "updateUserId": 2889,
+          "updateDt": "2015-03-24T11:35:11",
+          "versioning": 0,
+          "useType": "Maritime",
+          "vesselType": "Pleasure",
+          "aircraftType": "None",
+          "landUse": "None",
+          "note": "Some notes"
```

```suggestion
          "note": "Some notes",
          "modType": "Place Holder",
          "activationMode": "Automatic",
          "modVariant": "Operational"
```
👀 sorry I've just mapped all Uses fields so these ones will be there too after this PR: https://github.com/mcagov/beacons-etl/pull/3

Previously I thought maybe we should ask the Registry team about them, but actually these ones are pretty simple/straightforward